docs/proposals/20190709-cluster-spec-crds.md

```diff
@@ -208,7 +208,7 @@ When the cluster object is created, the cluster controller will retrieve the inf
 
 When an infrastructure object is updated, the provider controller will check the owner reference. If it is set, it will retrieve the cluster object to obtain the required cluster specification and starts the provisioning process. When the process finishes, it sets the `Infrastructure.Status.Ready` to true.
 
-When the cluster controller detects the `Infrastructure.Status.Ready` is set to true, it updates the Cluster status with information from `Infrastructure.Status` (e.g. `APIEndpoint`) and sets `Cluster.Status.InfrastructureReady` to true.
+When the cluster controller detects the `Infrastructure.Status.Ready` is set to true, it updates `Cluster.Status.APIEndpoint` from `Infrastructure.Status.APIEndpoint` and sets `Cluster.Status.InfrastructureReady` to true.
```

Comment:
These should be `APIEndpoints` given that's a slice of endpoints

Reply:
I'd almost like to see this switch from a slice of endpoints to a single endpoint for v1alpha2 given that clients can only consume a single endpoint today.

Reply:
💯

Reply:
That sounds good to me!